cfgov/jinja2/v1/blog/blog_page.html

```diff
@@ -18,7 +18,7 @@
     {% include 'article.html' %}
     <div class="block">
         <p>
-            Due to technical issues, the commenting feature of our blog is temporarily unavailable. We’re working to bring this functionality back, and look forward to hearing your feedback and comments about the CFPB’s work soon.
+            Join the conversation. Follow CFPB on <a class="icon-link icon-link__external-link" href="https://twitter.com/cfpb"><span class="icon-link_text">Twitter</span> </a> and <a class="icon-link icon-link__external-link" href="https://www.facebook.com/cfpb"><span class="icon-link_text">Facebook</span> </a>.
```

Comment:
I believe so. [See example in the social media molecule](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_includes/molecules/social-media.html#L74), @contolini.